mkosi

```diff
@@ -4646,6 +4611,13 @@ def load_args(args: CommandLineArguments) -> CommandLineArguments:
     if args.generated_root() and "bios" in args.boot_protocols:
         die("Sorry, BIOS cannot be combined with --minimize or squashfs filesystems")
 
+    if args.with_unified_kernel_images and args.distribution == Distribution.clear:
+        warn("Enabling --without-unified-kernel-images as Clear Linux does not support unified kernel images.")
```

Comment:
```suggestion
        warn("Using --without-unified-kernel-images as Clear Linux does not support unified kernel images.")
```
since it's actually disabling something